packages/components/src/Combobox/components/ComboboxContent/ComboboxContent.tsx

```diff
@@ -24,8 +25,8 @@ export function ComboboxContent(props: ComboboxContentProps): JSX.Element {
   const template = (
     <div
       ref={popperRef}
-      id="ATL-Combobox-Content"
-      data-testid="ATL-Combobox-Content"
+      id={COMBOBOX_MENU_ID}
```

Comment:
Combobox Menu should probably be modified to allow for multiple of them with different Ids since pages can have more than 1 combobox